### ISSUE TITLE
Issue with toggling a React template on and off

### ISSUE THREAD

**Issue by abhiaiyer91** [meteor/react-packages#177]
Hey guys,

I have a session variable toggling 2 react components on and off like so

``` handlebars
{{#if someVal}}
  {{> React component=reactComponentOne }}
{{else}}
  {{> React component=reactComponentTwo }}
{{/if}}
```

As these toggle I get exceptions in the console:

``` js
Error: Invariant Violation: unmountComponentAtNode(...): Target container is not a DOM element.
    at invariant (invariant.js:39)
    at Object.ReactMount.unmountComponentAtNode (ReactMount.js:609)
    at .<anonymous> (react-template-helper.js:77)
    at template.js:116
    at Function.Template._withTemplateInstanceFunc (template.js:457)
    at fireCallbacks (template.js:112)
    at .<anonymous> (template.js:218)
    at fireCallbacks (view.js:279)
    at Object.Tracker.nonreactive (tracker.js:589)
    at view.js:276
```

Any help?

**Comment by stubailo** [comment]
Sounds like this one to me: https://github.com/meteor/react-packages/pull/161

We haven't been able to publish a new version yet though.

**Comment by abhiaiyer91** [comment]
Ah okay, so wait for the new version? or wait for 1.3?

**Comment by stubailo** [comment]
Can you try updating to the newest `react-template-helper` that I just published and see if its fixed? version `0.2.6`?

**Comment by filipenevola** [comment]
I'm closing this just because it's too old. We can open new issues for items that are still valid.
